addons/html_builder/static/src/builder/snippets_menu_tabs/invisible_elements_tab.js

```diff
@@ -0,0 +1,104 @@
+import { Component, onWillStart, onWillUpdateProps, useState } from "@odoo/owl";
+import { WithSubEnv } from "../builder_helpers";
+import { getSnippetName } from "../../utils";
+export class InvisibleElementsTab extends Component {
+    static template = "html_builder.InvisibleElementsTab";
+    static components = { WithSubEnv };
+    static props = {
+        invisibleEls: { type: Array },
+        invisibleSelector: { type: String },
+    };
+
+    setup() {
+        this.state = useState({ invisibleEls: null });
+
+        onWillStart(() => this.updateInvisibleElementsPanel(this.props.invisibleEls));
+
+        onWillUpdateProps(async (nextProps) => {
+            this.updateInvisibleElementsPanel(nextProps["invisibleEls"]);
+        });
+    }
+
+    updateInvisibleElementsPanel(invisibleEls) {
```

Comment:
`createInvisibleElements` could suggest it is HTMLElements, and is not coherent with the terminology within the template. What about `updateInvisibleEntries` ?

Reply:
If you agree, I would go with `createInvisibleEntries` as we are "creating" the panel "from scratch" rather than "updating" information from it.